src/commands/init/init.ts

```diff
@@ -128,12 +128,23 @@ async function extractExample(name: string, dir: string, force?: boolean) {
         resp.body.pipe(extractor);
       });
 
-      const successLog = `Initialized "${chalk.bold(name)}" example in ${chalk.bold(toHumanPath(folder))}.`;
+      const successLog = `Initialized "${chalk.bold(
+        name
+      )}" example in ${chalk.bold(toHumanPath(folder))}.`;
       const folderRel = path.relative(process.cwd(), folder);
-      const deployHint = folderRel === ''
-        ? `To develop, run ${cmd('now dev')}. To deploy, run ${cmd('now')}.`
-        : `To develop, ${cmd(`cd ${folderRel}`)} and run ${cmd('now dev')}. To deploy, ${cmd(`cd ${folderRel}`)} and run ${cmd('now')}.`;
-      console.log(success(`${successLog} ${deployHint}`));
+      const developHint =
+        folderRel === ''
+          ? listItem(`To develop, run ${cmd('now dev')}.`)
+          : listItem(
+              `To develop, ${cmd(`cd ${folderRel}`)} and run ${cmd('now dev')}.`
+            );
+      const deployHint =
+        folderRel === ''
+          ? listItem(`To deploy, run ${cmd('now')}.`)
+          : listItem(
+              `To deploy, ${cmd(`cd ${folderRel}`)} and run ${cmd('now')}.`
+            );
+      console.log(success(`${successLog} \n ${developHint} \n ${deployHint}`));
```

Comment:
Why the spaces in between the `\n`'s?


```suggestion
      console.log(success(`${successLog}\n${developHint}\n${deployHint}`));
```